packages/framework/azure-client/src/AzureClient.ts

```diff
@@ -82,9 +85,9 @@ export class AzureClient {
             "/",
         );
         const createNewRequest = createAzureCreateNewRequest(
-            this.props.connection.orderer,
-            this.props.connection.storage,
-            this.props.connection.tenantId,
+            this.props.connection.endpoint,
+            this.props.connection.endpoint,
+            this.props.connection.tenantId ?? LOCAL_MODE_TENANT_ID,
         );
```

Comment:
Can you refactor this part too to reflect the single endpoint we need to pas around. You can also probably just change the one prop to be `AzureConnectionConfigType` and use the type property to pass it.

Reply:
I'm thinking `LOCAL_MODE_TENANT_ID` should be able to go away with this change.

Reply:
> Can you refactor this part too to reflect the single endpoint we need to pas around. You can also probably just change the one prop to be `AzureConnectionConfigType` and use the type property to pass it.

done 👍